### PR TITLE
Add SocialIcons@1.0.1 w/ git auto-update and its old version

### PR DESCRIPTION
**Pull request by maruilian11**
Pull request for issue: #9372 
@cdnjs/intern  @cdnjs/intern2 Please help me review this PR. Thanks!
0.1.0 is from https://github.com/jivinivan/soc.js/commit/7025a72e5ce47107a743786b66a410b482ebd3de

# Profile of the lib
 * Git repository (required): https://github.com/jivinivan/soc.js
 * Official website (optional, not the repository): https://ivansugerman.com/soc.js/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   Watch 5
   Star 72
   Fork 9
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@pvnr0082t thanks for your review.
@cdnjs/intern @cdnjs/intern2 please help me review this PR, thanks.

File: ajax/libs/SocialIcons/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "url": "https://github.com/jivinivan/soc.js.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jivinivan/soc.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "font/socicon-webfont*",
```
Comment (pvnr0082t) [inline ajax/libs/SocialIcons/package.json]: @maruilian11 I think we can use `font/**/*` here.

**Comment by maruilian11** [comment]
@pvnr0082t thanks for your review.
@cdnjs/intern @cdnjs/intern2 please help me review this PR, thanks.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
@maruilian11 I think you can use `add old version of SocialIcons` as your first line of second commit message, and `Manually add v0.1.0 by tracing commit log, cc #9372` as second line. Thanks!

**Comment by maruilian11** [comment]
@cdnjs/intern @cdnjs/intern2 I updated the commit msg. please help me review this PR. thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM

**Pull request merged**